packages/portal-proto/src/features/repositoryApp/FilesTable.tsx

```diff
@@ -77,6 +78,38 @@ const FilesTables: React.FC = () => {
   const allFilters = joinFilters(cohortFilters, repositoryFilters);
   const prevAllFilters = usePrevious(allFilters);
 
+  // TODO fix filters
```

Comment:
I'm not really sure what this comment is referring to so I just left it 🤔

Reply:
I'll see if I can figure out why and what it refers to.

Reply:
IIRC it's been there since a long time.

Reply:
looks like it can be removed, might be part of when i was building out the original functionality and had not fixed the filters yet